os-eap-launch/added/launch/resource-adapters-common.sh

```diff
@@ -106,7 +107,14 @@ function inject_resource_adapters_common() {
       resource_adapter="${resource_adapter}<transaction-support>$transaction_support</transaction-support>"
     fi
 
-    resource_adapter="${resource_adapter}<connection-definitions><connection-definition class-name=\"${ra_class}\" jndi-name=\"${ra_jndi}\" enabled=\"true\" use-java-context=\"true\">"
+    resource_adapter="${resource_adapter}<connection-definitions><connection-definition"
+
+    if [[ "$JBOSS_EAP_VERSION" == "7."* ]]; then
+      # monitor applications, look for unclosed resources.
+      local tracking=$(find_env "${ra_prefix}_TRACKING" "false")
```

Comment:
we should keep the default and set it only if the property is set.   (I believe in general it default to true, but perhaps not always)

Reply:
After discussion, I agree with @maschmid . Instead of checking the jboss eap version, just see if the tracking variable is set. If it is, set it, otherwise, do nothing. Additionally, instead of having to change all our KIE templates to set this variable to false, we can just set them in the image.yaml (not even needing to declare them in the templates).

Reply:
The problem of doing this is that it will also be applied on eap6, is this setting also necessary on eap6?

Reply:
The idea is that we wouldn't set the variable unless the en var is defined (so, don't use a default value when looking up the env var). It then wouldn't be applied to eap6, because no one would be setting the env var.